### PR TITLE
Summ 678

### PR DESCRIPTION
**Pull request by BenHoelker**

### CODE REVIEW

File: plugins/module_utils/network/awplus/argspec/acl/acl.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                                                                                         'send-to-mirror',
+                                                                                         'send-to-cpu']},
+                                                                  'destination_addr': {'type': 'str'},
+                                                                  'destination_port_protocol': {'elements': 'dict',
+                                                                                                'options': {'eq': {'type': 'int'},
+                                                                                                            'gt': {'type': 'int'},
+                                                                                                            'lt': {'type': 'int'},
+                                                                                                            'ne': {'type': 'int'},
+                                                                                                            'range': {'options': {'end': {'type': 'int'},
```
Comment (tony-vanderpeet) [inline plugins/module_utils/network/awplus/argspec/acl/acl.py:54]: Does this need elements: dict like the source port protocol?

Comment (BenHoelker) [reply plugins/module_utils/network/awplus/argspec/acl/acl.py:54]: Had a look in the model file, and I have forgotten to put 'elements: dict' in

File: playbooks/test_awplus_acl.yml [hunk 2]
```diff
@@ -0,0 +1,1037 @@
...
+          - acls:
+            - aces:
+                - source_addr: any
+                  protocols: ip
+                  action: deny
+              name: '72'
+              acl_type: standard
+            afi: IPv4
+        state: deleted
+
```
Comment (tony-vanderpeet) [inline playbooks/test_awplus_acl.yml:1033]: please remove blank lines at end of file.

**Review by tony-vanderpeet — COMMENTED** [review]
I left a few comments for you to look at. Overall, it all looks OK, but it's really too large to do an extensive review, so we'll be relying on the testing and the work the San Jose team will be doing with this to really iron out the issues.

**Review by tony-vanderpeet — COMMENTED** [review]
I really don't like the fact that you have added the same piece of code about 5 times. Similar to last time, why not bundle up that new code and the following command creation into a new subroutine? Also, please fix spelling of "their" while you're at it! :-)

**Review by tony-vanderpeet — APPROVED** [review]

**Pull request merged**